sql/src/main/java/io/crate/operation/collect/sources/InformationSchemaCollectSource.java

```diff
@@ -156,9 +161,12 @@ public void remove() {
     @SuppressWarnings("unchecked")
     public Collection<CrateCollector> getCollectors(CollectPhase collectPhase, RowReceiver downstream, JobCollectContext jobCollectContext) {
         Routing routing =  collectPhase.routing();
-        assert routing.locations().containsKey(TableInfo.NULL_NODE_ID);
-        assert routing.locations().get(TableInfo.NULL_NODE_ID).size() == 1;
-        String fqTableName = routing.locations().get(TableInfo.NULL_NODE_ID).keySet().iterator().next();
+        Map<String, Map<String, List<Integer>>> locations = routing.locations();
+        assert locations != null;
+        String localNodeId = clusterService.localNode().id();
```

Comment:
I think if this is really a problem we should do a follow up and somehow make sure that we avoid using clusterService.localNode().id() because currently it is actually used very often